runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -134,6 +117,17 @@ public class TimersPlugin extends Plugin
 	private int freezeTime = -1; // time frozen, in game ticks
 
 	private TimerTimer staminaTimer;
+	private TimerTimer antifireTimer;
+	private TimerTimer superAntifireTimer;
+	private TimerTimer magicImbueTimer;
+	private TimerTimer divineSuperAttackTimer;
+	private TimerTimer divineSuperStrengthTimer;
+	private TimerTimer divineSuperDefenceTimer;
+	private TimerTimer divineRangingTimer;
+	private TimerTimer divineMagicTimer;
+	private TimerTimer divineSuperCombatTimer;
+	private TimerTimer divineBastionTimer;
+	private TimerTimer divineBattlemageTimer;
 	private TimerTimer buffTimer;
 	private TimerTimer remedyTimer;
```

Comment:
Note to self, this may be better as a `Map<GameTimer, TimerTimer>` now that there's so many of these fields. Don't make this change right now--I'll add a commit to this PR doing that if it seems useful enough.